Mathlib/Tactic/NormNum.lean

```diff
@@ -138,14 +138,21 @@ def evalEq (α a b : Expr) : MetaM Expr := do
 end NormNum
 end Meta
 
+namespace Tactic
+
+open Lean.Parser.Tactic in
+syntax (name := normNum) "normNum" (" [" simpArg,* "]")? (ppSpace location)? : tactic
+
 open Meta Elab.Tactic in
-elab "normNum" : tactic =>
+elab_rules : tactic | `(normNum) => do
```

Comment:
```suggestion
elab_rules : tactic | `(tactic| normNum) => do
```
It is inferring the kind to `ident`, i.e. reading the syntax as a term `normNum` that doesn't exist and is maybe a local variable. @Kha Should the `: tactic` be used to influence the interpretation of syntax quotations in `inferMacroRulesAltKind`?

Reply:
It probably should, but it would require nontrivial changes in the parser.